src/main/java/org/apache/maven/plugins/changelog/ChangeLogReport.java

```diff
@@ -1218,9 +1224,13 @@ private void doChangedSetDetail(ChangeSet entry, Sink sink) {
         sink.tableCell_();
 
         sink.tableCell();
-        // doRevision( entry.getFiles(), bundle, sink );
-        doChangedFiles(entry.getFiles(), sink);
-        sink.lineBreak();
+
+        if (displayFileAndRevInfo) {
+            // doRevision( entry.getFiles(), bundle, sink );
```

Comment:
delete commented code

Reply:
Removed the commented code, thanks for spotting it